frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -101,6 +104,18 @@ const ProductAnalyticsOnboarding = (): JSX.Element => {
                         type: 'toggle',
                         inverseToggle: true,
                     },
+
+                    heatmapsEnabled
+                        ? {
+                              title: 'Enable heatmaps',
+                              description: `If you use our JavaScript libraries, we can capture general clicks, mouse movements,
+                               and scrolling to create heatmaps. 
+                               No additional events are created, and you can disable this at any time.`,
+                              teamProperty: 'heatmaps_opt_in',
+                              value: currentTeam?.heatmaps_opt_in ?? true,
```

Comment:
@raquelmsmith Does this actually save the value when they proceed? I couldn't quite find in the code if it will do that or not.
Like in the case that they just leave it selected

Reply:
Yep! It just works. (I pulled it down locally just to make sure 😄 )

It's saved here: https://github.com/PostHog/posthog/blob/89b329d4afe232c7980a0d781a0a3b7ba49ab1d7/frontend/src/scenes/onboarding/onboardingProductConfigurationLogic.ts#L50

Reply:
Ah cool thats where it is. Somehow I missed it - thanks!